tuned/plugins/plugin_scheduler.py

```diff
@@ -530,7 +531,15 @@ def _convert_sched_cfg(self, vals):
 		(scheduler, priority) = self._convert_sched_params(
 				scheduler, priority)
 		affinity = self._convert_affinity(affinity)
-		return (rule_prio, scheduler, priority, affinity, regex)
+		# split regex into two, one to match the cmdline, and one the comm
+		# if there is only one regex for the cmdline, then the comm is matched by ".*"
+		regexes = regex.split(":", 1)
```

Comment:
I would add possibility to escape colonn in regex if somebody would need ':' in process regex for some reason

Reply:
I have pushed an update that allows escaping ('\:')